services/blockchain-indexer/methods/dataService/controllers/network.js

```diff
@@ -28,6 +31,22 @@ const getNetworkStatus = async () => {
 	return networkStatus;
 };
 
+const getNetworkStatistics = async () => {
+	const peerStatistics = {
+		data: {},
+		meta: {},
+	};
+	const response = await dataService.getPeersStatistics();
+	if (response.data) peerStatistics.data = response.data;
+	if (response.meta) peerStatistics.meta = response.meta;
+
+	peerStatistics.data.totalLocked = await getTotalLocked();
+	peerStatistics.data.totalStaked = await getTotalStaked();
+
+	return peerStatistics;
```

Comment:
```suggestion
	const networkStats = {
		data: {},
		meta: {},
	};
	const response = await dataService.getPeersStatistics();
	if (response.data) networkStats.data = response.data;
	if (response.meta) networkStats.meta = response.meta;

	networkStats.data.totalLocked = await getTotalLocked();
	networkStats.data.totalStaked = await getTotalStaked();

	return networkStats;
```

Reply:
Cleaned up. This was because of WIP commit.